org.kde.skrooge.yml

```diff
@@ -12,13 +12,15 @@ finish-args:
   - --socket=x11
   - --socket=wayland
   - --device=dri
+  - --filesystem=host # needed for saving files
```

Comment:
Does this fix the saving file issue? Why not just having home filesystem access?

Reply:
Yes, this fixes the issue. Why host? Because security wise, it doesn't make a big difference and you can open files directly e.g. from USB drives.